testing/common/src/main/java/com/linecorp/centraldogma/testing/internal/CentralDogmaRuleDelegate.java

```diff
@@ -143,7 +143,12 @@ public final CompletableFuture<Void> startAsync(File dataDir) {
             configureClient(clientBuilder);
             configureClient(legacyClientBuilder);
 
-            final String accessToken = accessToken();
+            final String accessToken;
```

Comment:
> should this go before configureClient so that users have a chance to apply customization?

Oh, we should.

> Additionally, personally I feel like just using configureClient may be enough 

Somehow, I couldn't think of it. 😓 Fixed, thanks!